CommonCoreLegacy/src/main/java/com/skedgo/tripkit/routing/Trip.java

```diff
@@ -69,6 +69,12 @@ public class Trip implements ITimeRange {
     @SerializedName("shareURL")
     private String shareURL;
 
+    @SerializedName("subscribeURL")
+    private String subscribeURL;
+
+    @SerializedName("unsubscribeURL")
+    private String unsubscribeURL;
```

Comment:
Since the field names in your Java class are essentially identical to the corresponding names in the JSON data, it appears that there is no need to use the @SerializedName annotation. Is there any specific reason you included it?

Reply:
Hi @sg-jsonjuliane, since all of the fields for the API are setup like this, just did it for uniformity. 
Maybe we can have a review on this later on to set rules for future updates. Thank you 🙇